src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,99 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+
+// typography
+$pf-chart-global--FontSize--xs: $pf-global--FontSize--xs; // Axis Labels
+$pf-chart-global--FontSize--sm: $pf-global--FontSize--sm; // For Legend
+$pf-chart-global--FontSize--2xl: $pf-global--FontSize--2xl; // For Large Numbers
+
+// label
+$pf-chart-global-label--FontSize: $pf-chart-global--FontSize;
+$pf-chart-global-label--Padding: 10px;
+
+// Layout Props
+$pf-chart-global-layout--Padding: 50px;
+$pf-chart-global-layout--Height: 300px;
+$pf-chart-global-layout--Width: 450px;
+
+// Individual Charts
+
+// Area Chart
+$pf-chart-area--Opacity: .4;
+$pf-chart-area-stroke--Width: 2px;
+
+// Axis Chart
+$pf-chart-axis-axis-stroke--Width: 1px;
+$pf-chart-axis-axis-label--Padding: 100px;
+$pf-chart-axis--ticks--Width: 1px;  // this should be a var for size so not sure if width is appropriate
+
+// Bar Chart
+$pf-chart-bar--Width: 10px;
+
+// Box Plot Chart
+$pf-chart-boxplot-max--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-max-stroke--Width: 1px;
```

Comment:
Similar to my other comment https://github.com/patternfly/patternfly-next/pull/1863/files#r288707836

I think the prefix should be `$pf-chart-global` or `$pf-chart-boxplot` followed by `--concept` (or thing you're styling) followed by `--modifier` (if applicable), followed by `--PropertyCamelCase`

So this would become

`$pf-chart-boxplot--max--stroke--Width`
`$pf-chart-boxplot--median--stroke--Width`
`$pf-chart-boxplot--min--stroke--Width`
etc...

Reply:
Tried to follow this throughout the doc, but it would be great if you could double check the changes